app/utils/consts.ts

```diff
@@ -1,4 +1,10 @@
+import { version } from '../../package.json';
+
+const convertVersionStringToNumber = (versionString: string) => Number(versionString.replace(/[^\d]/g, ''));
+
 export default {
+    CURRENT_APP_DATA_VERSION: convertVersionStringToNumber(version),
```

Comment:
Minor:

I'm wondering about the name. Do we need the `DATA` part?

```suggestion
    CURRENT_APP_VERSION: convertVersionStringToNumber(version),
```

or even

```suggestion
    APP_VERSION: convertVersionStringToNumber(version),
```

both looks reasonable to me and are shorter

Reply:
Seems reasonable. I'll change it.

Reply:
Done.